README.md

```diff
@@ -1,19 +1,127 @@
 [Device42](http://www.device42.com/) is a Continuous Discovery software for your IT Infrastructure. It helps you automatically maintain an up-to-date inventory of your physical, virtual, and cloud servers and containers, network components, software/services/applications, and their inter-relationships and inter-dependencies.
 
-This repository has 2 different scripts that you can use with ansible.
-1. `d42_ansible_inventory_hostfile.py` can be used to create a static inventory file for ansible. You can group hosts by tags, customers, building or service level from Device42 data.
-2. `d42_ansible_dynamic_inventory.py` can be used to dynamically by ansible to get hosts from Device42 based on certain filters.
+This Ansible Collection contains the inventory and lookup plugins for Device42. 
+1. Inventory (device42.d42.d42) - can be used to dynamically generate Ansible inventory from Device42 devices.
+2. Lookup (device42.d42.d42)- This lookup plugin allows a playbook to fetch passwords or ip addresses for hosts from Device42.
+3. Lookup (device42.d42.d42_prompt) - This lookup plugin allows a playbook to fetch passwords or ip addresses for hosts from Device42 with prompt. 
 
+In the `contrib` directory you will find the legacy inventory scripts. Please favor the plugin over the legacy scripts:
+1. `contrib\inventory\d42_ansible_inventory_hostfile.py` can be used to create a static inventory file for ansible. You can group hosts by tags, customers, building or service level from Device42 data.
+2. `contrib\inventory\d42_ansible_dynamic_inventory.py` can be used to dynamically by ansible to get hosts from Device42 based on certain filters.
+
+-----------------------------
 ## Assumptions
+- This script works with Device42 12.0.0 and above
+### Requirements
+- ansible 2.9+
+- python 3.6.x+
+- requests (you can install it with pip install requests or apt-get install python-requests)
+- Ansible must have an available connection to your Device42 instance in order to collect devices for inventory
+
+### Installation
+#### Ansible Tower:
```

Comment:
For Ansible Tower, the preferred method to install a collection would be using a requirements file that pulls from [Ansible Galaxy](https://galaxy.ansible.com) (https://docs.ansible.com/ansible-tower/latest/html/userguide/projects.html#collections-support)

This both allows the user to manage via source control project and installs into the correct location for the auto plugin to find. I'm happy to help update the docs once we get closer to publishing the collection on galaxy.

Reply:
Hi Will,

We would appreciate you assistance in updating the docs for the Ansible Tower instructions for how to get the collection from Galaxy (and any other changes you think we should make to the docs) in preparation for deploying the collection to Galaxy.  Should we keep the manual installation instructions in addition to the Galaxy installation instructions?

Thanks,

Chris

Reply:
Here, I would leave a small section that reads something like `Using this collection in Ansible Tower` and then add a small section about including a requirements file under the collections directory in their scm. That file would look something like 
```
---
collections:
- device42.d42
```